frontend/src/components/Calendar/ClickableDateContainer.tsx

```diff
@@ -38,6 +39,7 @@ export const ClickableDateContainer = (props: { date: Date; eventPresent?: boole
   const history = useHistory()
   const { clickedDate, processedDates } = useSelector((state: RootState) => state.calendar)
   const assignedDateMonth = new Date(props.date.getFullYear(), props.date.getMonth(), props.date.getDate())
+  const params = useParams<{ facilityID: string; isEndDate: string }>()
```

Comment:
Camel case usage here would be better to be `facilityId` instead of `facilityID`